README.md

```diff
@@ -416,28 +416,28 @@ for how _we_ use it.
 To run benchmarks, you'll need to first build the STL, then build the benchmarks:
 
 ```cmd
-cmake -B out\x64 -S . -G Ninja
-cmake --build out\x64
-cmake -B out\benchmark -S benchmarks -G Ninja -DSTL_BINARY_DIR=out\x64
-cmake --build out\benchmark
+cmake -B out\build\x64 -S . -G Ninja

```

Comment:
Because they are not consistent within the paragraph (using `out\bench\benchmarks` and  `out\benchmark\benchmark`) and the preceding paragraph used `out\build\x64` which is also the default build location if you simply configure the cmake project inside VisualStudio

No strong preference from my side though

Reply:
I'd prefer to just change the `out\benchmark` -> `out\bench` without changing anything else.